Cypress/cypress/integration/Parallel_Tests/Create_Workflow/preDefinedWorkflow.spec.js

```diff
@@ -37,7 +39,23 @@ describe("Testing the workflow creation wizard using PreDefined Experiments", ()
 			0
 		);
 		cy.get("[data-cy=ControlButtons] Button").eq(1).click();
-		cy.wait(1000); // Needs to be removed with frontend enhancement
+		cy.wait(3000);
+		cy.get("table")
+			.find("tr")
+			.eq(1)
+			.then(($div) => {
+				cy.wrap($div).find("td").eq(0).should("contain.text", "pod"); // Matching Experiment
+			});
+		// Matching nodes of dagre graph
+		cy.get("[data-cy=DagreGraphSvg]")
+			.find("text")
```

Comment:
Can we give data-cy parameters to nodes?. That way, we will be more independent of the html tags.